<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventPopUp</class>
 <widget class="QDialog" name="eventPopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="rowWrapPolicy">
    <enum>QFormLayout::DontWrapRows</enum>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="eventName">
       <property name="text">
        <string>Name of Event:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>From:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="fromDate"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>To:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="toDate"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,0">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Category</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeWidget" name="category">
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="autoExpandDelay">
           <number>-1</number>
          </property>
          <property name="itemsExpandable">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>150</number>
          </attribute>
          <column>
           <property name="text">
            <string>Music</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Sports</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Food</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Leisure</string>
           </property>
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Hip-Hop</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Football</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Italian</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Beaches</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Rock</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Basketball</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Japaniese</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Bar Scene</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Country</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Baseball</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Thai</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
           <property name="text">
            <string>Reading</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Classic Rock</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="descriptionText"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Reocurring</string>
         </property>
         <widget class="QCheckBox" name="daily">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Daily</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="monthly">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>90</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Monthly</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="weekly">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>72</width>
            <height>19</height>
           </rect>
          </property>
          <property name="text">
           <string>Weekly</string>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>88</width>
            <height>21</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="yes">
             <property name="text">
              <string>Yes</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="no">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>eventPopUp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>eventPopUp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
